tests/test_smartbridge.py

```diff
@@ -154,7 +157,7 @@ async def fake_connect():
 
         self.target = smartbridge.Smartbridge(fake_connect)
 
-    async def initialize(self):
+    async def initialize(self, processor="Caseta"):
```

Comment:
Can we use constants or something instead of the string literal `"Caseta"`?

Reply:
I"ll need a bit of help here.  I didn't edit or use this test_smartbridge.py file.  How do I use/invoke it to evaluate any changes I make?  These edits were made by @johninaustin

Reply:
Ok, I've defined a constant DEFAULT_PROCESSOR and assigned it the string "Caseta"